app/src/main-process/main.ts

```diff
@@ -156,7 +156,13 @@ app.on('ready', () => {
 
   readyTime = now() - launchTime
 
-  app.setAsDefaultProtocolClient('x-github-client')
+   if (__DEV__) {
+    app.setAsDefaultProtocolClient('x-github-desktop-dev-auth')
+  } else {
+    app.setAsDefaultProtocolClient('x-github-desktop-auth')
+    app.setAsDefaultProtocolClient('x-github-client')
```

Comment:
We should always register for this protocol regardless of whether we're in dev or not